### PR TITLE
Added VM to ostypes to indicate KVM virtual machines

### PR DESCRIPTION
**Pull request by mbag**
Please review and merge this PR. It is related to issue erigones/esdc-ce#1

### CODE REVIEW

File: user-guide/monitoring/vm_monitoring.rst [hunk 1]
```diff
@@ -32,7 +32,7 @@ Dynamic Assignment of Monitoring Templates and Host Groups
 
 * **MON_ZABBIX_HOSTGROUPS_VM** - List of other existing Zabbix host groups, which will be used for all monitored servers in a virtual data center. Available placeholders are:
 
-    * ``{ostype}`` - ID type of the operating system (1 - *Linux*, 2 - *SunOS*, 3 - *BSD*, 4 - *Windows*, 5 - *SunOS Zone*).
+    * ``{ostype}`` - ID type of the operating system (1 - *Linux VM*, 2 - *SunOS VM*, 3 - *BSD VM*, 4 - *Windows VM*, 5 - *SunOS Zone*).
     * ``{ostype_text}`` - The type of the operating system with lowercase letters and spaces replaced with underscores (*linux*, *sunos*, *bsd*, *windows*, *sunos_zone*).
```
Comment (dn0) [inline user-guide/monitoring/vm_monitoring.rst]: The change in erigones/esdc-ce#1 also affects `vm.ostype_text()`, e.g. `bsd` will be `bsd_vm`

File: user-guide/monitoring/vm_monitoring.rst [hunk 2]
```diff
@@ -42,7 +42,7 @@ Dynamic Assignment of Monitoring Templates and Host Groups
 
 * **MON_ZABBIX_TEMPLATES_VM** - List of existing Zabbix templates, which will be used for all monitored servers in a virtual data center. Available placeholders are:
 
-    * ``{ostype}`` - ID type of the operating system (1 - *Linux*, 2 - *SunOS*, 3 - *BSD*, 4 - *Windows*, 5 - *SunOS Zone*).
+    * ``{ostype}`` - ID type of the operating system (1 - *Linux VM*, 2 - *SunOS VM*, 3 - *BSD VM*, 4 - *Windows VM*, 5 - *SunOS Zone*).
     * ``{ostype_text}`` - The type of the operating system with lowercase letters and spaces replaced with underscores (*linux*, *sunos*, *bsd*, *windows*, *sunos_zone*).
```
Comment (dn0) [inline user-guide/monitoring/vm_monitoring.rst]: See comment above.

**Review by dn0 — CHANGES_REQUESTED** [review]

File: user-guide/monitoring/vm_monitoring.rst (hunk 1 above)
Comment (mbag) [reply user-guide/monitoring/vm_monitoring.rst]: Made changes in 75195cb9123b67a1711e9d44395746a3016eaf8a

File: user-guide/monitoring/vm_monitoring.rst (hunk 2 above)
Comment (mbag) [reply user-guide/monitoring/vm_monitoring.rst]: Made requested changes in 75195cb9123b67a1711e9d44395746a3016eaf8a

**Review by dn0 — APPROVED** [review]

**Pull request merged**